spec/brainstem/presenter_spec.rb

```diff
@@ -430,6 +430,16 @@ def workspace.synthetic_id
 
         result = presenter.group_present(Workspace.order('id desc'), %w[tasks user lead_user tasks_with_lambda])
       end
+
+      it "includes any nested hash associations declared via the preload DSL directive" do
```

Comment:
It'd be fine to make this into a separate test that asserts correct dedeuping across symbols and strings.